src/viur/core/skeleton.py

```diff
@@ -1760,8 +1760,7 @@ class RefSkel(RelSkel):
     @classmethod
     def fromSkel(cls, kindName: str, *args: list[str]) -> t.Type[RefSkel]:
         """
-            Creates a relSkel from a skeleton-class using only the bones explicitly named
-            in \*args
+            Creates a relSkel from a skeleton-class using only the bones explicitly named in args.
```

Comment:
Or code inline formatting?
```suggestion
            Creates a relSkel from a skeleton-class using only the bones explicitly named in ``args``.
```